lib/page/redirect-next-page/redirect-next-page.js

```diff
@@ -149,6 +153,28 @@ module.exports = async function getRedirectForNextPage (pageInstance, userData)
       }
     }
 
+    const checkNextPage = getInstance(getNextPage(currentPage, userData)._id)
+
+    // if the next page has upload components
+    if (hasUploadComponents(checkNextPage)) {
+      // And the next page has a conditionality.
```

Comment:
what is conditionality in this context?

Reply:
The next page has the visibility if the answer of the page is "yes" go to the next page (the "show" property of the page.